src/components/task-app/task-app.component.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import TaskForm from "../task-form";
+import TaskList from "../task-list";
+
+const TaskApp = () => {
+  const [toggleForm, setToggleForm] = useState(false);
```

Comment:
sorry for being picky! I would suggest updating this state to `formVisibility`, as `toggleForm` sounds more like an action rather than a state:

```typescript
  const [formVisibility, setFormVisibility] = useState(false);
```

Reply:
05758ea3ca9475a61e11905fead2370794d39d14 Variable name changed as advised.